doc/source/whatsnew/v1.5.3.rst

```diff
@@ -30,6 +30,7 @@ Bug fixes
 - Bug in :meth:`.Styler.to_excel` leading to error when unrecognized ``border-style`` (e.g. ``"hair"``) provided to Excel writers (:issue:`48649`)
 - Bug when chaining several :meth:`.Styler.concat` calls, only the last styler was concatenated (:issue:`49207`)
 - Fixed bug when instantiating a :class:`DataFrame` subclass inheriting from ``typing.Generic`` that triggered a ``UserWarning`` on python 3.11 (:issue:`49649`)
+- Typing: add :attr:`.Timestamp.nanosecond` to :class:`Timestamp`
```

Comment:
This is only internal, we don't need a whatsnew for this

Reply:
Is it internal though? If users run a typechecker and have an `#ignore` comment everywhere they use the `nanosecond` property, they won't get informed that they can possibly remove those.